pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.hivemq</groupId>
     <artifactId>hivemq-extension-archetype</artifactId>
-    <version>4.3.0</version>
+    <version>4.3.3</version>
```

Comment:
the version should be 4.3.1 as this has nothing to do with the current HiveMQ patch version